docs/ref/contrib/admin/index.txt

```diff
@@ -906,13 +910,17 @@ subclass::
 
                   def lookups(self, request, model_admin):
                       if request.user.is_superuser:
-                          return super(AuthDecadeBornListFilter,
-                              self).lookups(request, model_admin)
+                          return super(
```

Comment:
I talked to @kutenai here at the sprints and recommended this formatting; it doesn't bother me, and I think it's an improvement over the existing formatting. But if you (Tim) feel it's inconsistent with Django practice, I'll defer.